js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
It is not dev.parity either :)

It is actually the version of parity.js that gets used by all dapps running inside Parity. e.g. https://github.com/gavofyork/gavcoin/blob/master/index.html#L11